### PR TITLE
create Calculator UI

### PR DESCRIPTION
**Pull request by ram1117**
In this PR, I have implemented the following. Please review and provide feedback. Thank you.

- Implement calculator UI using React functional components.
- Removed boilerplate files and images from `src` directory
- Fix linter errors

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,11 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Calculator />
+    </>
```
Comment (andgarzonmal) [inline src/App.js:8]: - [ ] Please according to project requirements take into account your App.js component should be class-based

image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <Button value="1" />
+      <Button value="2" />
+      <Button value="3" />
+      <Button value="+" className="yellow" />
+    </>
+    <>
+      <Button value="0" className="zero-button" />
+      <Button value="." />
+      <Button value="=" className="yellow" />
+    </>
```
Comment (andgarzonmal) [inline src/components/Calculator.js:36]: - [ ] please remember that the `Calculator.js` component should be class-based

image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @ram1117✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/App.js (hunk 1 above)
Comment (ram1117) [reply src/App.js:8]: Hi @andgarzonmal the project requirement states we use functional components.

image

And some students get approval for using functional components. Seems the project requirements have been updated recently

image

**Comment by andgarzonmal** [comment]
# ‼️ Please approve this project ‼️

 Everything is ok on this project,  the curriculum has changed some days ago and the reviewer guidelines are no up to date. 
 as it stays on the new project requirements students should use FUNCTIONAL COMPONENTS instead of Class components 

image

**Review by andgarzonmal — APPROVED** [review]
Hi @ram1117 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**